src/theory/arith/theory_arith_private.cpp

```diff
@@ -1933,7 +1933,7 @@ void TheoryArithPrivate::outputConflicts(){
         }
 
         Assert(conflict.getNumChildren() == pf.d_farkasCoefficients->size());
-        if (confConstraint->hasSimpleFarkasProof())
+        if (confConstraint->hasSimpleFarkasProof() && confConstraint->getNegation()->isPossiblyTightenedAssumption())
```

Comment:
Is this formatted?

Reply:
Ah! It was not. Good catch. Fixed in be32fcb48.